CONTRIBUTORS.md

```diff
@@ -0,0 +1,12 @@
+# Contributors to the Parsec project
+
```

Comment:
Taking example from the [TF-A project](https://github.com/ARM-software/arm-trusted-firmware/blob/a04808c16cfc126d9fe572ae7c4b5a3d39de5796/docs/about/acknowledgements.rst), it would be nice to add here:

> This file is to acknowledge the specific contributors referred to in "Contributors of the Parsec project" copyright notices.

to make it clear what this file is for.